docs/source.rst

```diff
@@ -156,17 +210,16 @@ below message.
 |Delete received messages|
 
 If the server experiences a large number of new sources signing up at
-once and is overloaded with submissions, the journalist will flag your
-message on their end and you will see the message below. They can't
-write a reply to you until you've seen this message for security
-reasons. This will only happen the first time a journalist replies and
-with subsequent replies you will skip this step. Click **Refresh** or
-log in again to see if a journalist has responded.
+once and is overloaded with submissions, you may see the message below:
```

Comment:
I propose changing overloaded to "experiences a surge in traffic" or "is experiencing high traffic volume"

Reply:
Yeah I can see why :), reworded in 53e978e3a3ee5bfbdb81c2f144c00b4f60edb73b.